tox.ini

```
@@ -124,15 +124,15 @@ commands =
     cov: coverage run --parallel-mode \
     !cov: python \
         -m pytest -v --color=yes --basetemp={envtmpdir} --ignore napari/_vispy \
-        --ignore napari/_qt --ignore napari/_tests --ignore tools \
+        --ignore src/napari/_qt --src/ignore napari/_tests --ignore tools \
```

```suggestion
        --ignore src/napari/_qt --ignore src/napari/_tests --ignore tools \
```
I think this is what you meant.